vcpkg/get_started/get-started-msbuild.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Install and use packages with MSBuild in Visual Studio
+description: Tutorial guides the user through the process of installing and using packages with MSBuild and Visual Studio.
+author: data-queue
```

Comment:
```suggestion
zone_pivot_group_filename: zone-pivot-groups.json
zone_pivot_groups: operating-systems
author: data-queue
```

This will ensure the zone pivots work when you include the "Install vcpkg" steps. We may need to make changes to the "Install vcpkg" since this article is essentially Windows only. As of now, I don't think there is a way to only include some of the zone pivots and not the others.